granary/bluesky.py

```diff
@@ -782,6 +782,34 @@ def from_as1(obj, out_type=None, blobs=None, aspects=None, client=None,
     # attachments to embed(s), including quoted posts
     record_embed = record_record_embed = external_embed = external_record_embed = None
 
+    # convert link at end of post to a quote
+    attachment_urls = [att['url'] for att in attachments if 'url' in att]
+    
+    text_end = len(full_text)
+
+    for tag in link_tags:
+      # check that the link is at the end and the text is the url
+      if (tag.get('objectType') != 'link' or tag.get('url') != tag.get('displayName') or
```

Comment:
This checks that the link text is the same as the `href`, right?
(I'm not familiar with the API, so just making sure.)